templates/init/dataconnect/connector.yaml

```diff
@@ -9,5 +9,7 @@ authMode: "PUBLIC"
 #     packageJsonDir: < Optional. Path to your Javascript app's package.json>
 #   swiftSdk:
 #     outputDir: <Path where you want the generated SDK to be written to, relative to this file>
+#     package: "firebasegen/default"
 #   kotlinSdk:
 #     outputDir: <Path where you want the generated SDK to be written to, relative to this file>
+#     package: com.connectors.default
```

Comment:
Please get rid of the `com.` prefix for the kotlin packge; just `connectors.default`